routes/index.js

```diff
@@ -5,8 +5,14 @@ const Queries = require( '../database/queries.js' )
 const pgPromise = require('pg-promise')
 
 // /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get('/', function( req, res ) {
+  Tasks.getAll().then(Tasks => {
```

Comment:
When I see identifiers that begin with a capital letter, I think it is a class name.  Best practice is to name types (classes) beginning with capital letters, and identifiers beginning with lower case capital letters.  Also, I think your catch block is associated with the incorrect block - it should be on the promise chain, not the call to `render`
```js
Tasks.getAll()
  .then( tasks => {
    response.render( 'index', { title: 'To Do List', tasks })
  })
  .catch( error => next( error ) )
```